### PR TITLE
Update attachPVC controller to use getPreferredVersion

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Another

Depends on web console [PR 161](https://github.com/openshift/origin-web-common/pull/161)

@spadgett

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
Rebased & added view.

File: app/scripts/controllers/attachPVC.js [hunk 1]
```diff
@@ -67,6 +67,10 @@ angular.module('openshiftConsole')
       subpage: 'Add Storage'
     });
 
+    $scope.pvcVersion = APIService.getPreferredVersion('persistentvolumeclaims');
+    $scope.quotaVersion = APIService.getPreferredVersion('resourcequotas');
+    $scope.appliedQuotaVersion = APIService.getPreferredVersion('appliedclusterresourcequotas');
```
Comment (spadgett) [inline app/scripts/controllers/attachPVC.js]: If we're going to shorten the name, I prefer `clusterQuotaVersion`. That makes it more obvious it's cross namespace.

Comment (spadgett) [reply app/scripts/controllers/attachPVC.js]: I'm good with `clusterQuotaVersion`

**Comment by benjaminapetersen** [comment]
rebased & updated vars to full name.

**Pull request merged**